css/styles.css

```diff
@@ -0,0 +1,848 @@
+  height: 6rem;
+  position: absolute;
+  top: 1.5rem;
+  left: 1rem;
+}
+
+.news-item__price--span {
+  display: block;
+  text-transform: uppercase;
+  color: var(--grey-1);
+  font-weight: 300;
+  font-size: 1.4rem;
+}
+
+.news-item__list {
+  margin-top: 1rem;
+  border-bottom: 0.1rem solid var(--Red);
+  margin-bottom: 1rem;
+}
+
+.news-item__list .fas {
+  margin-right: 0.5rem;
+  color: var(--black);
+}
+
+.news2-item__list li {
+  list-style: disc;
+  color: var(--Red);
+  margin-bottom: 0.8rem;
+  border-bottom: 1px solid #eee;
+  padding-bottom: 1rem;
+}
+
+.top-footer__social li {
+  background-color: var(--grey-1);
+  height: 4rem;
+  width: 4rem;
+  border-radius: 50%;
+  margin: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.top-footer__links li {
+  margin: 1rem;
+}
+
+.news__links li:not(:last-child) {
+  border-right: 1px solid var(--Red);
+  padding-right: 1.2rem;
+}
+
+.light-bg {
+  background-color: var(--faint-yellow);
+}
+
+.news-item__text {
+  padding-bottom: 1rem;
+}
+
+.news2 {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 2.5rem;
+}
+
+.news2-item__img {
+  width: 100%;
+  background-position: top left;
+  background-size: cover;
+  height: 30rem;
+  position: relative;
+}
+
+.news2--1 {
+  background: var(--image-overlay), url(../img/FB-Avatar-Featured-368x276.jpg);
+}
+
+.news2--2 {
+  background: var(--image-overlay), url(../img/sale_28070_primary_image.jpg);
+}
+
+.news2--3 {
+  background: var(--image-overlay), url(../img/sale_27875_primary_image.jpg);
+}
+
+.news2--4 {
+  background: var(--image-overlay), url(../img/podium.png);
+}
+
+.news2--5 {
+  background: var(--image-overlay), url(../img/creative.png);
+}
+
+.news2--6 {
+  background: var(--image-overlay), url(../img/sale_28070_primary_image.jpg);
+}
+
+.news2-item__latest--img {
+  width: 9.6rem;
+  height: 9.6rem;
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.news2__heading {
+  font-size: 2rem;
+  letter-spacing: 0.15rem;
+}
+
+.news2-item__latest {
+  color: var(--Red);
+  background-color: var(--light-pink);
+  padding: 1.3rem;
+  line-height: 1.2;
+  position: relative;
+  margin-top: 1px;
+  margin-bottom: 0.5rem;
+  height: 9.6rem;
+  overflow: hidden;
+}
+
+.news2-item__latest--title {
+  width: 70%;
+  font-size: 1.4rem;
+}
+
+.news2-item__latest--title span {
+  display: block;
+  text-transform: uppercase;
+  font-weight: 300;
+}
+
+.news2-item__list {
+  padding: 1.5rem;
+}
+
+.news2-item__list--links {
+  color: #333;
+  font-size: 1.7rem;
+}
+
+.checked {
+  text-decoration: line-through;
+}
+
+.price {
+  font-size: 1.6rem;
+  color: var(--grey-1);
+}
+
+.footer {
+  background-color: #131313;
+  height: 30vh;
+  display: flex;
+  flex-flow: column;
+  align-items: center;
+  padding-bottom: 2rem;
+}
+
+.top-footer {
+  width: 100%;
+  padding: 4.2rem 0 2.8rem;
+  background-color: var(--grey-2);
+}
+
+.top-footer__social {
+  display: flex;
+  flex-flow: row;
```

Comment:
It seems the social links in the footer are not aligning on a new row when on small screens
`flex-flow: row wrap;` --> should fix it

Reply:
Yes it did 👍 
Thank you. 💯 🙏